lib/src/signed_video_h26x_sign.c

```diff
@@ -590,10 +590,30 @@ signed_video_set_recurrence_interval(signed_video_t *self, int recurrence)
 
   svi_rc status;
   SVI_TRY()
+    SVI_THROW_IF_WITH_MSG(recurrence < RECURRENCE_ALWAYS, SVI_NOT_SUPPORTED,
+        "Recurrence interval < RECURRENCE_ALWAYS");
     SVI_THROW_IF_WITH_MSG(
-        recurrence < RECURRENCE_MIN, SVI_NOT_SUPPORTED, "Invalid recurrence interval");
+        recurrence > MAX_GOP_LENGTH, SVI_NOT_SUPPORTED, "Recurrence interval > MAX_GOP_LENGTH");
```

Comment:
This is wrong. MAX_GOP_LENGTH is number of frames per GOP. Recurrence counts in GOPs, not in frames.
Further, I don't think there should be an upper limit. There could be applications where it is super long, and still would work on the validation side, because all streams get the first SEI.